pyseir/deployment/demographic_mapper.py

```diff
@@ -341,22 +347,41 @@ def _reconstruct_hospital_recovery_inflow_rates(self):
 
         hospital_recovery_inflow_rates = {}
         hospital_recovery_inflow_rates["HGen"] = (
-            1 - mortality_inflow_rates["HGen"]
-        ) / self.parameters["hospitalization_length_of_stay_general"]
+            1 / self.parameters["hospitalization_length_of_stay_general"] - mortality_inflow_rates["HGen"])
```

Comment:
Calculation is corrected since mortality_inflow_rates have already taken into account length of stay.